imgs/magic-wormhole-mailbox/default.nix

```diff
@@ -0,0 +1,37 @@
+{ system ? builtins.currentSystem }:
+let
+  nwi = import ../../nwi.nix;
+  pkgs = import ../../pin { snapshot = "nixos-20-03_0"; };
+  lib = pkgs.lib;
+  importedPythonPkgs = with pkgs; python37.withPackages (pythonPkgs: with pythonPkgs; [ magic-wormhole-mailbox-server ]);
+  contents = with pkgs; [ bash coreutils procps importedPythonPkgs ];
+in
+pkgs.dockerTools.buildImage {
+  inherit contents;
+  name = "nebulaworks/magic-wormhole-mailbox-server";
+  # Doesnt matter will use the derivation
+  # when publishing to registry
+  tag = "latest";
+  extraCommands = ''
+    # make sure /tmp exists
+    mkdir -m 1777 tmp
+
+    # Magic-Wormhole-Mailbox Directory
+    mkdir -m 1777 magic-mailbox
```

Comment:
Can we make this more uniform? `/var/lib/wormhole` or something like that would be more appropriate I think. This way we can standardize on the hierarchy of the filesystem: https://www.man7.org/linux/man-pages/man7/hier.7.html vs just having a bunch of things hanging off `/`.

Also lets make the something a tighter, `750` seems more appropriate